source/includes/_advanced-usage.md

```diff
@@ -0,0 +1,38 @@
+#Advanced usage
+
+## Opening product dialog programatically
+
+Tipser product dialog can be openend programatically by using the underlying [Tipser SDK](#tipser-sdk) instance.
+
+__If you use Tipser Elements:__
+
+```javascript
+const elements = tipser.elements(posId, config);
+elements.sdkInstance.openProductDialog(productId);
+```
+
+__If you use Tipser React Elements:__
+
+1. Create a custom React component and mount it anywhere under `TipserElementsProvider` in React elements hierarchy.
+2. Inside that component you can use tipserSdk instance from the context `TipserContext`. This how it can be done with [React Hooks](https://reactjs.org/docs/hooks-intro.html):
+
+
+```javascript
+import { TipserContext } from `@tipser/tipser-elements`;
+
+const DialogOpener = () => {
+  const context = useContext(TipserContext());
+  conts sdk = context.data.tipserSdk;
+  
+  useEffect(() => { sdk.openProductDialog(productId); }, []);
```

Comment:
```suggestion
  const productId = "5d0245b09bee660001745f07";
  useEffect(() => { sdk.openProductDialog(productId); }, []);
```

Reply:
It's a bit off the good practices of react hooks - lack of `productId` in array but I guess that's okay;

I would prefer not to have `productId` coming from the void. It always struck me "where I should get it from?" when I see something like that in other products docs.

Reply:
I think the names like `posIId`, `config`, `productId` are pretty self descriptive and the customers who know our library enough to reach this far corner of our docs will have no trouble understanding it. However I will add a short sentence explaining how to get product ids. Is it okay?

Reply:
> It's a bit off the good practices of react hooks - lack of productId in array but I guess that's okay

Overall you are of course right. But in this specific example I think we just want to open a dialog once, no matter what (we don't want our component to keep opening dialogs if `productId` gets changed for some unknown reasons?)

Reply:
@chmurson does it look clearer now?

![image](https://user-images.githubusercontent.com/1180737/77148821-98d2ad00-6a90-11ea-9ba6-18b1d0917d78.png)